src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -552,7 +553,7 @@ def retry_jobs(self, job_id_list: List[str]) -> List[dict]:
         for job_id in error_ids:
             retry_results.append(
                 {
-                    "job": {"state": {"job_id": job_id, "status": "does_not_exist"}},
+                    "job": get_error_output_state(job_id),
                     "error": "does_not_exist",
```

Comment:
We might consider making `"does_not_exist"` a constant, as it seems to get used a bunch. Not a big deal here, though.